boards/xtensa/m5stack_core2/m5stack_core2.dts

```diff
@@ -0,0 +1,254 @@
+/*
+ * Copyright (c) 2023 Martin Kiepfer <mrmarteng@teleschirm.org>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/dts-v1/;
+
+#include <espressif/esp32/esp32_common.dtsi>
+#include "m5stack_core2-pinctrl.dtsi"
+#include "grove_connectors.dtsi"
+#include <zephyr/dt-bindings/display/ili9xxx.h>
+#include <zephyr/dt-bindings/regulator/axp192.h>
+
+/ {
+	model = "esp32";
+	compatible = "espressif,esp32";
+
+	aliases {
+		pwr-led = &pwr_led;
```

Comment:
it's confusing (but not blocking either I guess) that blinky wouldn't compile. Could led0 be an alias for the power LED? I couldn't seem to get it to work though but I am not really familiar with MFD (I would have expected the AXP192 GPIOs to be "just like" any other GPIO?)

Reply:
A `gpio-leds` driver is required for the Blinky example. The `axp192_gpio1` definition within the gpio driver is required to make sure the LED is initialised during boot up (via gpis-hogs).
I added the gpio-leds definition in the latest commit. The blinky example should work now.